docs/how-to-write-function.md

```diff
@@ -67,7 +67,13 @@ Similar to a Contract using `Ledger` object to manage assets, a Function uses `D
 
 ### How Functions and Contracts are tied together
 
-Functions to be executed are specified in a contract argument with `__functions__` JSON key. As described in [the contract guide](how-to-write-contract.md#write-a-complex-contract), a contract can invoke another contract, so multiple contracts and multiple functions can be grouped together.
+Functions to be executed are specified in a contract argument in a JSON format with `__functions__` as a key and an array of function IDs as a value as follows:
```

Comment:
I don't fully know this page is the best place to write details about the function, but developers might want to know the registration of functions and correspondence of params (e.g., function ID/name) through the same example on this page. For example, how about adding the following?

---

First functions must be registered to execute with contracts as follows:
```
register-function --properties client.properties --function-id test-function --function-binary-name com.example.function.PaymentFunction --function-class-file PaymentFunction.class
```

Reply:
Thank you!
As pointed out, there is not really "how to use" guide anywhere, so let me add something like that including how to register.